aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -272,6 +286,57 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    // Offer rotation capability of this account to another address
```

Comment:
Use three slash /// comments for top-level function comments. I would write account as `account`, to more clearly indicate the variable representing the account.

Reply:
Also, explain a little what the other three args are.

Reply:
ok makes sense - will add more comments